storm-core/src/clj/backtype/storm/daemon/supervisor.clj

```diff
@@ -326,16 +331,29 @@
                                          (log-error t "Error when processing event")
                                          (exit-process! 20 "Error when processing an event")
                                          ))
+   :blob-update-timer (mk-timer :kill-fn (fn [t]
+                                           (log-error t "Error when processing blob-update")
+                                           (exit-process! 20 "Error when processing a blob-update")
```

Comment:
Do we want a different error code for blob-update?
Maybe not, I'm not sure. The error message is different than the others.